src/data/agile-quiz.ts

```diff
@@ -217,6 +219,20 @@ const agileQuiz = [
       "In Agile, Timeboxing is a set amount of time for a planned activity to take place. This technique is often used in project management to help with productivity for projects. ",
     Link: "https://en.wikipedia.org/wiki/Timeboxing",
   },
+  {
+    Question: "In Agile, what are Story Points?",
+    Answer:
+      "An estimate of the amount of effort required to deliver a user story",
+    Distractor1:
+      "An estimate of the amount of sprints required to deliver a user story",
+    Distractor2:
+      "An estimate of the amount of days required to deliver a user story",
+    Distractor3:
+      "An estimate of the amount of hours required to deliver a user story",
+    Explanation:
+      "Story points are units of measure for expressing an estimate of the overall effort required to fully implement a product backlog item or any other piece of work.",
```

Comment:
Let's reword this so it is not as similar to this Atlassian article 
https://www.atlassian.com/agile/project-management/estimation#:~:text=Story%20points%20are%20units%20of,work%2C%20and%20risk%20or%20uncertainty.

```suggestion
      "Story points are used to estimate the amount of effort required to deliver a user story for a project.",
```